src/main/cpp/concurrent_map.h

```diff
@@ -421,6 +421,11 @@ struct Migration : public JobCoordinator::Job {
 		Source() : table(nullptr), index(0) {
 		}
 
+#ifdef C_ATOMICS
```

Comment:
What happens if C_ATOMICS aren't defined? Does the code even still compile with the missing copy constructor?